### PR TITLE
LIVY-358. [Follow-up] Add unit test to verify header size configurations

### PR DESCRIPTION
**Pull request by jerryshao**
This is a followup of #329 to add unit test to verify header size configurations. CC @zjffdu .

### CODE REVIEW

**Comment by zjffdu** [comment]
Thanks @jerryshao , sorry I miss to review one thing in #329,  could add those 2 configuration in livy.conf.template ?

**Comment by jerryshao** [comment]
Thanks, I will update the conf file.

File: client-http/src/test/scala/com/cloudera/livy/client/http/LivyConnectionSpec.scala [hunk 1]
```diff
@@ -60,31 +60,29 @@ class LivyConnectionSpec extends FunSpecLike with BeforeAndAfterAll with LivyBas
...
+        .setScheme(server.protocol)
+        .setHost(server.host)
+        .setPort(server.port)
         .setUserInfo(URLEncoder.encode(username, utf8Name), URLEncoder.encode(password, utf8Name))
         .build()
       info(uri.toString)
       val conn = new LivyConnection(uri, new HttpConf(null))
       try {
-        conn.get(classOf[Boolean], "/") shouldBe true
+        conn.get(classOf[Object], "/") should not be (null)
```
Comment (zjffdu) [inline client-http/src/test/scala/com/cloudera/livy/client/http/LivyConnectionSpec.scala]: why is this changed ?

Comment (jerryshao) [inline client-http/src/test/scala/com/cloudera/livy/client/http/LivyConnectionSpec.scala]: Because previously we use this [servlet](https://github.com/cloudera/livy/pull/331/files#diff-6ca014ee7958eb376596cb519952a394L63) which will return `Boolean`. Now we change to use `InteractiveSessionServlet`, so using `Boolean` will meet JSON parse error.

**Comment by zjffdu** [comment]
LGTM

**Comment by jerryshao** [comment]
@zjffdu any further comment?
